connection.js

```diff
@@ -1,11 +1,13 @@
-import { Sequelize } from 'sequelize';
+import Sequelize from 'sequelize';
 
-// Option 3: Passing parameters separately (other dialects)
-const sequelize = new Sequelize('postgres://postgres:qod@123@localhost:5432/user'); // Example for postgres
+const sequelize = new Sequelize('UserInformation', 'user_1', 'test123', {
+  host: 'localhost',
```

Comment:
Use env file for config, don't hardcode it.

Reply:
ok

Reply:
ok.